plugins/maven/maven3-server-impl/src/org/jetbrains/idea/maven/server/Maven3XServerEmbedder.java

```diff
@@ -137,6 +140,15 @@ public abstract class Maven3XServerEmbedder extends Maven3ServerEmbedder {
 
   @NotNull private final MavenImporterSpy myImporterSpy;
 
+  private static final Set<String> TYCHO_BANNED_PACKAGING = new HashSet<>(Arrays.asList(
+    MavenTychoConstants.PACKAGING_ECLIPSE_FEATURE,
+    MavenTychoConstants.PACKAGING_ECLIPSE_REPOSITORY,
+    MavenTychoConstants.PACKAGING_ECLIPSE_APPLICATION,
+    MavenTychoConstants.PACKAGING_ECLIPSE_UPDATE_SITE,
+    MavenTychoConstants.PACKAGING_ECLIPSE_TARGET_DEFINITION,
+    MavenTychoConstants.PACKAGING_P2_INSTALLABLE_UNIT
+  ));
```

Comment:
Why banning those?
The `eclipse_target_definition` packaging allows to define platform wide dependencies (that can be then resolved from the OSGI entries in the `MANIFEST.MF` file).

Reply:
This one might actually be related to how I'm using Tycho.
In my case the dependencies are simply pulled in by p2 repositories, I have no Target definition.